.storybook/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../tsconfig.json",
+  "exclude": ["../**/test.ts", "../**/*.spec.ts"],
+  "include": ["../libs/**/*.ts"]
```

Comment:
Does order make difference here?
It seems like what is excluded could be a small set of what is included. probably not any more after all the specs move out of `libs/` folder.

Reply:
I don't _believe_ so.. my understanding is that TS will use those arrays to create the more complex globs it uses to find files - so either way it's still creating the same inclusion/exclusion.